src/api-reference/direct-connects/hotel-service-2/Use-cases.markdown

```diff
@@ -4,17 +4,20 @@ layout: reference
 ---
```

Comment:
Add the pre-release include from the doc template, we'll add it to all the Hotel Service 2 pages. 
{% include prerelease.html %}